torch/_inductor/mkldnn.py

```diff
@@ -599,7 +599,8 @@ def mkldnn_fuse_fx(gm: torch.fx.GraphModule, example_inputs):
     # why re-run fuse_unary? we want to enable conv+binary+unary fusion,
     # such as conv+add+relu for vision model.
     gm = fuse_unary(gm)
-    gm = pack_module(gm)
+    if torch._inductor.config.mkldnn_weight_prepack:
```

Comment:
Can you put the config under "config.cpp"? Also suggest to name it `weight_prepack` since weight prepacking improves performance with the cost of larger memory footprint in general, not specific to mkldnn.